docker/Dockerfile

```diff
@@ -0,0 +1,42 @@
+# Dockerfile to setup neo-python on the master branch.
+# https://github.com/CityOfZion/neo-python/tree/master/docker
+#
+# Building (these commands create a Docker image called neopython):
+#
+#    $ docker build -f Dockerfile -t neopython .
+#
+# Build without caching:
+#
+#    $ docker build --no-cache -f Dockerfile -t neopython .
+#
+# At this point there is a Docker image called 'neopython'. If you want to run it connecting
+# to a private network, make sure the privatenet container is already running.
+# https://hub.docker.com/r/cityofzion/neo-privatenet
+#
+# Start a container interactively, opening a bash in `/neo-python`, and mounting the current directory as `/neo-python/sc`:
+#
+#    $ docker run --rm -it -v $(pwd):/neo-python/sc --net=host -h neopython --name neopython neopython /bin/bash
+#
+# Once you are inside the container, you can start neo-python with `python3 prompt.py -p` (using -p to connect to a private net).
+# To update neo-python, just run `git pull` and `pip install -e .`
+FROM ubuntu:16.04
+
+# Install dependencies
+RUN apt-get update && apt-get install -y \
+    wget \
+    git-core \
+    python3-dev \
```

Comment:
@metachris Do we need to define this as Python 3.5?

Reply:
yes